package.json

```diff
@@ -41,6 +43,7 @@
   },
   "devDependencies": {
     "assume": "1.4.x",
+    "assume-sinon": "^0.0.0",
```

Comment:
You don't need this since you're not using any of the `assume-sinon` extensions.